### ISSUE TITLE
Issues on labs with LabView with IIT Delhi , IIT Kanpur, IIT GUWAHATI labs

### ISSUE THREAD

**Issue by pushpdeep** [Virtual-Labs/engineers-forum#55]
We are mapping the experiments that needs porting to Sandhi from Labview. During the check we found these errors. Please help resolve the errors asap.
IIT DELHI
1.] Virtual Wireless labs
    Unable to download the labview experiments. It gives the error as shown in fig. 1
2.] Engineering & Electromagnetic lab
       Unable to open the link . It gives the error as shown in fig. 2

IIT KANPUR
1.] Virtual labs oscillations
         Unable to locate the experiment file for Labview.
2.] Ultra-fast laser spectroscopy
         Unable to locate the experiment file for Labview.
3.] Transducer & Instrumentation virtual lab
         Unable to login it gives the error as shown in fig. 3
4.].RF & Microwave characterization lab.
           Unable to locate the experiment file for Labview.

IIT GUWAHATI
1.] Virtual laboratory experience in fluid and thermal sciences
        Unable to Click on the link for Virtual laboratory Experience in Fluid and Thermal Sciences. It redirects the page to Virtual Labs At IIT GUWAHATI.
2.] Electronic Instrumentation.
       Unable to locate the experiment file for Labview. It gives the error as shown in fig. 4
3.] Virtual mass transfer lab.
          Unable to click on the experiment file for Labview.

**Comment by ecthiender** [comment]
We can't see any figures! :) Maybe the Sandhi team can help on this?

**Comment by sanchita-dixit** [comment]
@pushpdeep If this issue was resolved, kindly close this issue. If you need further help, kindly comment.

**Comment by ecthiender** [comment]
@Virtual-Labs/ie-iitd-team @Virtual-Labs/ie-iitk-team @Virtual-Labs/ie-iitg-team  Please comment

**Comment by sumitdahiyavlab** [comment]
Currently, At IIT Delhi, We have prioritized 5 labs for integration purpose that are as follows:

- Queueing Networks Modelling Lab
- Bioreactor Modeling and Simulation Laboratory
- Virtual Smart Structures and Dynamics Laboratory
- Engineering Electro-magnetics Laboratory
- Power Electronics and Electric Drives Laboratory

Out of these, Only Engineering Electro-magnetics Laboratory uses Lab View. Whose conversion process is in pipeline.

So, I want to comment on the two points raised by Pushdeep.

1.] Virtual Wireless labs
Unable to download the labview experiments. It gives the error as shown in fig. 1

Response to 1.] I will ask the respective Lab Developers. After getting reply, I will respond to this point.

2.] Engineering & Electromagnetic lab
Unable to open the link . It gives the error as shown in fig. 2

Response to 2.]  Regarding links, An issue is raised on engineers forum. Issue No. #171. Once it is done. I hope the links would work fine.

**Comment by sumitdahiyavlab** [comment]
@pushpdeep Could you please send/show me the figures associated with the errors, which you have mentioned above. It would be helpful for me to understand the errors.

**Comment by sumitdahiyavlab** [comment]
@pushpdeep @ecthiender 

Hi All,

I would like to add some more comments regarding this issue.

Regarding Virtual Wireless Lab, I talked with the concerned authority. They don't find any such errors in the lab. If you find it, then please report the errors in some convenient way. It would help us to easily find and correct the errors.

In case of Engineering Electro-magnetics Laboratory, recently the lab is updated on vlab.co.in. You could check it now, if you find any problem then please reply.

**Comment by bijudas45** [comment]
Please send the figures associated with the errors for better understanding.

**Comment by ecthiender** [comment]
@pushpdeep request you to comment.

**Comment by sumitdahiyavlab** [comment]
Hi Everyone,

I am facing some issues in Engineering Electro-magnetics laboratory IITD.

The issue is as follows: 

1. when i open any experiment e.g. I opened experiment 1 as shown in fig below:

![1](https://cloud.githubusercontent.com/assets/11206668/9159871/c2c4dcae-3f4e-11e5-98ad-0fe6fdf98e2b.png)

2. As soon as i click on the experiment link in above figure, this message appears:

![2](https://cloud.githubusercontent.com/assets/11206668/9159882/dfa970be-3f4e-11e5-81e2-aaf8db4a8ef0.PNG)

3. Although Application.exe is present at its designated place:

![3](https://cloud.githubusercontent.com/assets/11206668/9159907/5ad75788-3f4f-11e5-8743-e48289345043.PNG)

4. The html code of exp1.html is also pointing at right place: 

![5](https://cloud.githubusercontent.com/assets/11206668/9159920/a4b5b14c-3f4f-11e5-9504-0fdabd05d153.PNG)

Still the Application.exe is not downloading.

Also, when i open application.exe on github. This message appears:
![6](https://cloud.githubusercontent.com/assets/11206668/9159933/d6aa9ba4-3f4f-11e5-9807-bd403d77f3ce.PNG)

This is happening for majority of the experiments in the lab. Kindly provide your suggestions.

Github Address of the lab: https://github.com/Virtual-Labs/engineering-electro-magnetics-laboratory-iitd

Address of the lab on cloud: http://eem-iitd.vlabs.ac.in/

**Comment by pushpdeep** [comment]
Dear Anon,
I am not aware of exact issue.
may be if some domain expert  can look into it.?

Thanks

Regards



*---Pushpdeep.Mishra*


*Tuberculosis Daily http://bit.ly/iCACcq <http://bit.ly/iCACcq>*

**Comment by ecthiender** [comment]
@pushpdeep I was requesting you to comment, because @bijudas45 and @sumitdahiyavlab wanted figures/screenshots regarding the errors you have mentioned in their lab.

The request to comment was not regarding the above issue.

**Comment by ecthiender** [comment]
@sumitdahiyavlab the issue you have raised, is a separate issue and is not related to the current issue. Please create a new issue and move the above issue you are facing into that new issue. We will comment there.

**Comment by sumitdahiyavlab** [comment]
@ecthiender Ok. Thanks for your feedback. I am creating a new issue for the above problem  (regarding Engineering Electro-magnetics laboratory) and removing it from here.
